consoles/serial_screen.pm

```diff
@@ -231,8 +222,7 @@ C<{ matched => 0, string => 'text from the terminal' }>
 on failure.
 
 =cut
-sub read_until {
-    my ($self, $pattern, $timeout) = @_[0 .. 2];
+sub read_until ($self, $pattern, $timeout) {
     my $fd       = $self->{fd_read};
     my %nargs    = @_[3 .. $#_];
```

Comment:
This line must go into the signature as well, otherwise the function only expects 3 arguments.

Reply:
done